langchain/vectorstores/chroma.py

```diff
@@ -80,6 +80,8 @@ def __init__(
         else:
             if client_settings:
                 self._client_settings = client_settings
+                if persist_directory is not None:
```

Comment:
hm if someone is provide client_settings do we want to be overwriting them? if someone wants to specify client_setting and persist_directory should we actually enforce that persist_directory is added to client_settings before being passed in (e.g. we'd raise an error here)